llvm/lib/CodeGen/ExpandFp.cpp

```diff
@@ -74,11 +74,63 @@ class FRemExpander {
   /// Constant 1 of type \p ExTy.
   Value *One;
 
+  /// The frem argument/return types that can be expanded by this class.
+  // TODO The expansion could work for other floating point types
+  // as well, but this would require additional testing.
+  inline static const SmallVector<MVT, 3> ExpandableTypes{MVT::f16, MVT::f32,
+                                                          MVT::f64};
```

Comment:
```suggestion
 static constexpr std::array<MVT, 3> ExpandableTypes{MVT::f16, MVT::f32,
                                                          MVT::f64};
```

Reply:
Or just regular c array

Reply:
> Or just regular c array

I changed it to the "constexpr std::array" variant.